ibm_mq/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG - IBM MQ
 
+## 4.0.0 / 2020-03-11
+
+* [Fixed] IBM MQ refactor. See [#5902](https://github.com/DataDog/integrations-core/pull/5902).
+* [Changed] Add configuration option for the Channel Definition API version. See [#5905](https://github.com/DataDog/integrations-core/pull/5905).
```

Comment:
> * [Changed] Add configuration option for the Channel Definition API version. See [#5905](https://github.com/DataDog/integrations-core/pull/5905).

Should that be "Added" ? 

Since the library is currently using mqcd_version 6 by default, this should be a no-op for existing users.